source/MRMesh/MRMovementBuildBody.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+#include "MRMeshFwd.h"
+#include "MRVector3.h"
+#include <optional>
+
+namespace MR
+{
+
+struct MovementBuildBodyParams
+{
+    /// if this flag is set, rotate body in trajectory vertices
+    /// according to its rotation
+    /// otherwise body movement will be done without any rotation
+    bool allowRotation{ true };
+    /// point around which body is rotated (if allowRotation)
+    /// if not set body bounding box center is used
+    std::optional<Vector3f> rotationCenter;
+    /// facing direction of body, used for initial rotation (if allowRotation)
+    /// if not set body accumulative normal is used
+    std::optional<Vector3f> bodyNormal;
+    /// optional transform trajectory space to body space
+    const AffineXf3f* t2bXf{ nullptr };
```

Comment:
do we really need it? The user can transform either the input contours, or transform resulting mesh